executor/ddl_test.go

```diff
@@ -211,6 +212,24 @@ func (s *testSuite) TestCreateDropTable(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testSuite) TestCreateDropView(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("create or replace view drop_test as select 1,2")
+	_, err := tk.Exec("drop view if exists drop_test")
+	c.Assert(err, IsNil)
+
+	_, err = tk.Exec("drop view mysql.gc_delete_range")
+	c.Assert(err.Error(), Equals, "Drop tidb system table 'mysql.gc_delete_range' is forbidden")
+
+	_, err = tk.Exec("drop view drop_test")
+	c.Assert(err.Error(), Equals, "[schema:1051]Unknown table 'test.drop_test'")
+
+	tk.MustExec("create table t_v(a int)")
+	_, err = tk.Exec("drop view t_v")
+	c.Assert(err.Error(), Equals, "[schema:1347]'test.t_v' is not VIEW")
```

Comment:
Please add a test about drop multiple views in a statement.

Reply:
already have `drop view v1,v2,v3,v4,v5,v6`, but I add test it's error examiation.